pymc3/distributions/discrete.py

```diff
@@ -295,7 +296,7 @@ class NegativeBinomial(Discrete):
     def __init__(self, mu, alpha, *args, **kwargs):
         super(NegativeBinomial, self).__init__(*args, **kwargs)
         self.mu = mu = tt.as_tensor_variable(mu)
-        self.alpha = alpha = tt.as_tensor_variable(alpha)
```

Comment:
some of these had a point that below where we construct the self.mean we want that to be a theano expression too. It's only safe to remove this if we use self.alpha below too. And that wasn't the case always.